jupyter-notebook/Dockerfile

```
@@ -51,6 +51,7 @@ RUN python3 -m pip install --no-cache-dir -r /etc/skel/requirements_template.txt
     mkdir /home/techuser/.jupyter && chown techuser /home/techuser/.jupyter
 
 COPY jupyter_lab_config.py /home/techuser/.jupyter/jupyter_lab_config.py
+COPY jupyter_terminal_config.json /home/techuser/.jupyter/lab/user-settings/@jupyterlab/terminal-extension/plugin.jupyterlab-settings
```

```suggestion
COPY  --chown=techuser jupyter_terminal_config.json /home/techuser/.jupyter/lab/user-settings/@jupyterlab/terminal-extension/plugin.jupyterlab-settings
```

Otherwise the techuser doesn't have write permissions on the file, which will lead to 500 Internal Server Errors when the configuration is changed via the UI.